### DISCUSSION TITLE
Adding style for specific Column.

### DISCUSSION THREAD

**Discussion by mena234** [TanStack/table#4177] [Q&A]
I'm using react-table v7.8.0, I wonder how to add some style to a specific Column.

For example, let's say I have a table with ( first name - last name - age ), I need to add some style for the last name column only.

**Comment by LinnJS** [comment]
Hey @mena234 I kinda answered this question in this other discussion https://github.com/TanStack/table/discussions/4191

There is a meta property on ColumnsDef [here](https://tanstack.com/table/v8/docs/api/core/column-def#meta) and you can pass a `className` via `meta` and put it on only that column to color it or constrain the width or something like that.
